### DISCUSSION TITLE
Arduino_GFX: how do I push an image from a pre-made .h file?

### DISCUSSION THREAD

**Discussion by Salam797** [moononournation/Arduino_GFX#429] [Q&A]
how do I push an image from a pre-made .h file?

**Comment by moononournation** [comment]
You may check the [ImgViewerPROGMEM.ino](https://github.com/moononournation/Arduino_GFX/blob/master/examples/ImgViewer/ImgViewerPROGMEM/ImgViewerPROGMEM.ino) example.
